docs/Using Fleet/MDM-OS-updates.md

```diff
@@ -26,6 +26,10 @@ When a minimum version is enforced, the end users see a native macOS notificatio
 
 If the host was turned off when the deadline passed, the update will be scheduled an hour after it’s turned on.
 
+For macOS (14.0+) devices that use Automated Device Enrollment (ADE), if the device is below the specified
```

Comment:
```suggestion
For macOS devices that use Automated Device Enrollment (ADE), if the device is below the specified
```

Reply:
I think we already have subtitle that covers experience for macOS < 14 below, so I would just cut version from here.